docs/dev/build_android.md

```diff
@@ -5,54 +5,121 @@ This article describes how to build OpenVINO for Android operating systems.
 ## Software requirements
 
 - [CMake](https://cmake.org/download/) 3.13 or higher
-- Android NDK (this guide has been validated with r20 release)
+- [SCons](https://scons.org/pages/download.html) 4.6.0 or higher
+- [Android SDK Platform Tools](https://developer.android.com/tools/releases/platform-tools) (this guide has been validated with 30.0.0 release)
+- [Android NDK](https://developer.android.com/ndk/downloads) (this guide has been validated with r26 release)
 
 ## How to build
 
-1. Download and unpack [Android NDK](https://developer.android.com/ndk/downloads). Let's assume that `~/Downloads` is used as a working folder.
+### Create work directory and set environment variables for building
   ```sh
-  cd ~/Downloads
-  wget https://dl.google.com/android/repository/android-ndk-r20-linux-x86_64.zip
-
-  unzip android-ndk-r20-linux-x86_64.zip
-  mv android-ndk-r20 android-ndk
+  mkdir openvino-android
+  export OPV_HOME_DIR=${PWD}/openvino-android
+  export CURRENT_ANDROID_ABI=arm64-v8a
```

Comment:
can we highlight that x86_64 can be used as well? E.g. that value to set depending on underlying HW

Reply:
@ilya-lavrenov updated, but below is also a description of which ABI can be used